app/controllers/tips_controller.rb

```diff
@@ -56,4 +56,12 @@ def set_owner
     @owner = @tip.user
   end
 
+  def set_city_for_tip
```

Comment:
Is this the `tips#set_city_for_tip` controller action, or a private controller instance method?